migration-guide/redhat-to-ibm.adoc

```diff
@@ -320,7 +320,7 @@ The update script creates a backup folder in the extracted `bamoe-$VERSION-updat
 +
 For files that already exist in your {PRODUCT} distribution, such as .`jar` files for the decision engine or other add-ons, replace the existing version of the file with the new version from the {PRODUCT_SUPPORT_PAGE}[IBM Support] page.
 
-. If you use the standalone {PRODUCT_LONG} v{VERSION_80} - Maven Repository artifact (`bamoe-8.0.3-maven-repository.zip`), such as in air-gap environments, download I{PRODUCT_LONG} v{VERSION_80} - Maven Repository and extract the downloaded `bamoe-8.0.3-maven-repository.zip` file to your existing `~/maven-repository` directory to update the relevant contents.
+. If you use the standalone {PRODUCT_LONG} v{VERSION_80} - Maven Repository artifact (`bamoe-8.0.3-maven-repository.zip`), such as in air-gap environments, download{PRODUCT_LONG} v{VERSION_80} - Maven Repository and extract the downloaded `bamoe-8.0.3-maven-repository.zip` file to your existing `~/maven-repository` directory to update the relevant contents.
```

Comment:
here I would have a question about `8.0.3` mention, as on the weekly meetings we mention `8.0.5` if I am not wrong, not sure who could help with our understanding here?

Reply:
It should probably use the latest 8.x release.

Reply:
Created a new {VERSION_8_LATEST} attribute, which we can set to 8.0.5 when it's officially ready

Reply:
Thank you for this!